README.md

```diff
@@ -10,17 +10,18 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
 ## Dependencies
 
 The only direct dependencies of this code are OpenSSL 1.0.2 or higher or 1.1.0 or higher and
-[aws-c-common](https://github.com/awslabs/aws-c-common) v0.3.0 or higher. You will also need
+[aws-c-common](https://github.com/awslabs/aws-c-common) v0.3.15. You will also need
```

Comment:
It would be nice if we could master these pinned versions in a single place. I'm not sure what that single place should be, though.

Reply:
I second that, we could have a file called `upstream-dependencies` and pull in the versions we require for all builds from there.

Reply:
Totally agree; from testing to building docker containers- having one place to look would help; probably belongs in a new issue ?

Reply:
This work belongs in a separate issue.